jira/client.py

```diff
@@ -1093,9 +1093,16 @@ def create_issues(self, field_list, prefetch=True):
             data['issueUpdates'].append(issue_data)
 
         url = self._get_url('issue/bulk')
-        r = self._session.post(url, data=json.dumps(data))
-
-        raw_issue_json = json_loads(r)
+        try:
+            r = self._session.post(url, data=json.dumps(data))
+            raw_issue_json = json_loads(r)
+        # Catching case where none of the issues has been created. See https://github.com/pycontribs/jira/issues/350
+        except JIRAError as je:
+            if je.status_code == 400:
+                raw_issue_json = json.loads(je.response.text)
+            else:
+                raise
+        print(raw_issue_json)
```

Comment:
A library should never use `print()`. If needed for debug, please use logging instead, at debug level, and only when there is an error.